SimPEG/EM/Analytics/__init__.py

```diff
@@ -2,3 +2,4 @@
 from FDEM import hzAnalyticDipoleF
 from FDEMcasing import *
 from DC import DCAnalyticHalf, DCAnalyticSphere
+from FDEM_fields import *
```

Comment:
are all of the functions in this file specific to dipoles? Perhaps we could name the file `FDEMDipolarFields.py` or something along those lines